pkg/api/client.go

```diff
@@ -530,3 +531,77 @@ type GetEnvResponse struct {
 	IsArchived bool       `json:"isArchived"`
 	ArchivedAt *time.Time `json:"archivedAt"`
 }
+
+type EvaluateTemplateRequest struct {
+	// Value is an arbitrary value that can include one or more Template values.
+	// Each Template will be evaluated, and if successful, will be replaced in
+	// Value with its output. The updated Value will be returned in the response.
+	// If any templates fail to evaluate, the Template will be left in Value
+	// and a separate error will be returned in the response.
+	Value       interface{}                   `json:"value"`
+	RunID       string                        `json:"runID"`
+	Env         Env                           `json:"env"`
+	Resources   map[string]resources.Resource `json:"resources"`
+	Configs     map[string]string             `json:"configs"`
+	ParamValues map[string]interface{}        `json:"params"`
```

Comment:
```suggestion
	ParamValues map[string]interface{}        `json:"paramValues"`
```

Unless there's a really good reason for this to be `params`, best to be consistent here.